src/__test__/components/Header.test.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import initialExperimentState, { experimentTemplate } from 'redux/reducers/experiments/initialState';
+import { render, screen } from '@testing-library/react';
+
+import Header from 'components/Header';
+
+import createTestComponentFactory from '__test__/test-utils/testComponentFactory';
+
+jest.mock('next-seo');
+jest.mock('components/UserButton', () => () => <>User Button</>);
+
+const mockStore = configureMockStore([thunk]);
+
+const mockExperimentName = 'mock experiment';
+const mockExperimentId = 'abcd1234';
+
+const storeState = mockStore({
+  ...initialExperimentState,
+  experiments: {
+    [mockExperimentId]: {
+      ...experimentTemplate,
+      name: mockExperimentName,
+    },
+  },
+});
```

Comment:
I see this is a brand-new test. Why writing it with the fake store instead of using the real store directly?